utils/print_bingo_card_image.py

```diff
@@ -1,85 +1,50 @@
-    center_column_one = 170
-    center_column_two = 450
-    center_column_three = 720
-
-    #Define the center (y-axis) of each row
-    center_row_one = 290
-    center_row_two = 525
-    center_row_three = 751
-
-    #spacing between rows in px
-    spacing_rows = 12
-
-    #printing text on the right spot
-    draw.text(((center_column_one - px_removal_to_center(song_1)), (center_row_one - spacing_rows)), song_1, fill="black", font=font)
-    draw.text(((center_column_one - px_removal_to_center(artist_1)), (center_row_one + spacing_rows)), artist_1, fill="black", font=font)
-    draw.text(((center_column_one - px_removal_to_center(song_2)), (center_row_two - spacing_rows)), song_2, fill="black", font=font)
-    draw.text(((center_column_one - px_removal_to_center(artist_2)), (center_row_two + spacing_rows)), artist_2, fill="black", font=font)
-    draw.text(((center_column_one - px_removal_to_center(song_3)), (center_row_three - spacing_rows)), song_3, fill="black", font=font)
-    draw.text(((center_column_one - px_removal_to_center(artist_3)), (center_row_three + spacing_rows)), artist_3, fill="black", font=font)
-
-    draw.text(((center_column_two - px_removal_to_center(song_4)), center_row_one - spacing_rows), song_4, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(artist_4)), center_row_one + spacing_rows), artist_4, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(song_5)), (center_row_two - spacing_rows)), song_5, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(artist_5)), (center_row_two + spacing_rows)), artist_5, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(song_6)), (center_row_three - spacing_rows)), song_6, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(artist_6)), (center_row_three + spacing_rows)), artist_6, fill="black", font=font)
-
-    draw.text(((center_column_three - px_removal_to_center(song_7)), center_row_one - spacing_rows), song_7, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(artist_7)), center_row_one + spacing_rows), artist_7, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(song_8)), (center_row_two - spacing_rows)), song_8, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(artist_8)), (center_row_two + spacing_rows)), artist_8, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(song_9)), (center_row_three - spacing_rows)), song_9, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(artist_9)), (center_row_three + spacing_rows)), artist_9, fill="black", font=font)
-
-    image_rotated = image.rotate(90)
+import textwrap
+
+def make_bingo_card(image_path, df, card_size=9):
+    image_path = r'n:\My Documents\Julien\iQuizzen\music_bingo\scraps\template.jpg'
```

Comment:
Does iQuizz still has the posibility to adjust the bingocard or does this make it fixxed?

Reply:
Goed gezien. Ik had die regel er even tussen gezet om iets te testen (en dan e.e.a. te selecteren, runnen i.p.v. de main.py te runnen). Ik verwijder die regel.